### PR TITLE
Add rmt_size_id to create rebins

### PR DESCRIPTION
**Pull request by miramanz**
Summary of this change:
Add rmt_size_id to create rebins

### Added
- . rmt_size_id to create rebins

How to test this code via the UI:
Production | Runs | List Production Runs | rebins | create rebin

### CODE REVIEW

File: lib/raw_materials/interactors/rmt_bin_interactor.rb [hunk 1]
```diff
@@ -273,7 +273,7 @@ def create_rebin_groups(production_run_id, params) # rubocop:disable Metrics/Abc
         params.delete(:qty_bins_to_create)
```
Comment (jcsjcs) [inline lib/raw_materials/interactors/rmt_bin_interactor.rb:273]: This serves no purpose.

File: lib/raw_materials/interactors/rmt_bin_interactor.rb [hunk 2]
```diff
@@ -273,7 +273,7 @@ def create_rebin_groups(production_run_id, params) # rubocop:disable Metrics/Abc
...
         bin_asset_numbers.map(&:last).each do |bin_asset_number|
           params[:bin_asset_number] = bin_asset_number
```
Comment (jcsjcs) [inline lib/raw_materials/interactors/rmt_bin_interactor.rb:275]: This serves no purpose. Does creating the bin using `res` do what it needs to do? The code differs from before using params.

**Review by jcsjcs — CHANGES_REQUESTED** [review]

**Comment by lmatoti** [comment]
Create Rebins
rmt_size_id added

File: lib/raw_materials/interactors/rmt_bin_interactor.rb [hunk 3]
```diff
@@ -270,7 +268,6 @@ def create_rebin_groups(production_run_id, params) # rubocop:disable Metrics/Abc
 
       created_rebins = []
       repo.transaction do
-        params.delete(:qty_bins_to_create)
         bin_asset_numbers.map(&:last).each do |bin_asset_number|
           res.to_h[:bin_asset_number] = bin_asset_number
```
Comment (jcsjcs) [inline lib/raw_materials/interactors/rmt_bin_interactor.rb:272]: This doesn't do anything useful. It should be combined with the next line of code if the asset number needs to be saved.

**Review by jcsjcs — CHANGES_REQUESTED** [review]

**Pull request merged**